Makefile

```diff
@@ -135,6 +143,30 @@ local-cross: bin/skopeo.darwin.amd64 bin/skopeo.linux.arm bin/skopeo.linux.arm64
 build-container:
 	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -t "$(IMAGE)" .
 
+# Build container specific to arch
+build-arch-container:
+	${CONTAINER_RUNTIME} build -t "${ARCH_IMAGE}" contrib/skopeoimage/${SOURCE_TYPE}
+
+# Push container specific to arch
+push-arch-container:
+	echo "${QUAY_PASSWORD}" | ${CONTAINER_RUNTIME} login ${REGISTRY} -u "${QUAY_USERNAME}" --password-stdin
+	${CONTAINER_RUNTIME} push "${ARCH_IMAGE}"
+ifneq ($(EXTRA_REPO),)
+	${CONTAINER_RUNTIME} tag "${ARCH_IMAGE}" "${EXTRA_ARCH_IMAGE}"
+	${CONTAINER_RUNTIME} push "${EXTRA_ARCH_IMAGE}"
+endif
+
+# Push manifest to make multi-arch image
+push-ma-manifest:
+	echo "${QUAY_PASSWORD}" | docker login ${REGISTRY} -u "${QUAY_USERNAME}" --password-stdin
+	sudo chmod 0755 /etc/docker
```

Comment:
??!

Why is it even necessary?

It’s really not OK to risk that this will happen without prompting and to someone who runs this more or less by accident.

Reply:
that was a bug for docker https://github.com/moby/moby/pull/37847
it is fixed in later version, but docker `18.06.0-ce` still has it

Reply:
Thanks; that _definitely_ needs a comment, and it’s again a thing I’d much prefer to push out of the `Makefile` to `.travis.yml` or a a Travis-specific script.

Reply:
In the last version I used not default latest Ubuntu(focal) as Travis env, and it actually resolved this bug.